google/cloud/aiplatform/datasets/dataset.py

```diff
@@ -100,35 +122,23 @@ def create(
                 is defined as an OpenAPI 3.0.2 Schema Object. The schema files
                 that can be used here are found in gs://google-cloud-
                 aiplatform/schema/dataset/metadata/.
-            gcs_source: Optional[Sequence[str]]=None:
+            gcs_source (Optional[Union[str, Sequence[str]]]):
                 Google Cloud Storage URI(-s) to the
                 input file(s). May contain wildcards. For more
                 information on wildcards, see
                 https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames.
-            bq_source: Optional[str]=None:
+                examples:
+                    str: "gs://bucket/file.csv"
+                    Sequence[str]: ["gs://bucket/file1.csv", "gs://bucket/file2.csv"]
+            bq_source (Optional[str]):
                 BigQuery URI to the input table.
```

Comment:
We may also want a sample for `bq_source` to match the documentation for `gcs_source`.